container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -327,119 +321,6 @@ func (manager *KubernetesManager) GetIngressesByLabels(ctx context.Context, name
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "An unexpected error occurred retrieving the list of Kubernetes nodes.")
-	} else if len(nodes.Items) > 1 {
-		return nil, stacktrace.NewError("Using persistent volumes on Kubernetes with multiple nodes is currently " +
-			"not supported. Reach out to Kurtosis if you need this feature.")
-	}
-
-	hostPathPath := path.Join(volumeHostPathRootDirectory, namespace, volumeName)
-	hostPathType := apiv1.HostPathDirectoryOrCreate
-	persistentVolumeDefinition := apiv1.PersistentVolume{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "",
-			APIVersion: "",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:            volumeName,
-			GenerateName:    "",
-			Namespace:       "",
-			SelfLink:        "",
-			UID:             "",
-			ResourceVersion: "",
-			Generation:      0,
-			CreationTimestamp: metav1.Time{
-				Time: time.Time{},
-			},
-			DeletionTimestamp:          nil,
-			DeletionGracePeriodSeconds: nil,
-			Labels:                     labels,
-			Annotations:                nil,
-			OwnerReferences:            nil,
-			Finalizers:                 nil,
-			ManagedFields:              nil,
-		},
-		Spec: apiv1.PersistentVolumeSpec{
-			Capacity: apiv1.ResourceList{
-				apiv1.ResourceStorage: *resource.NewQuantity(requiredSize, resource.BinarySI),
-			},
-			PersistentVolumeSource: apiv1.PersistentVolumeSource{
-				GCEPersistentDisk:    nil,
-				AWSElasticBlockStore: nil,
-				HostPath: &apiv1.HostPathVolumeSource{
-					Path: hostPathPath,
-					Type: &hostPathType,
-				},
-				Glusterfs:            nil,
-				NFS:                  nil,
-				RBD:                  nil,
-				ISCSI:                nil,
-				Cinder:               nil,
-				CephFS:               nil,
-				FC:                   nil,
-				Flocker:              nil,
-				FlexVolume:           nil,
-				AzureFile:            nil,
-				VsphereVolume:        nil,
-				Quobyte:              nil,
-				AzureDisk:            nil,
-				PhotonPersistentDisk: nil,
-				PortworxVolume:       nil,
-				ScaleIO:              nil,
-				Local:                nil,
-				StorageOS:            nil,
-				CSI:                  nil,
-			},
-			AccessModes: []apiv1.PersistentVolumeAccessMode{
-				apiv1.ReadWriteOnce, // ReadWriteOncePod would be better, but it's a fairly recent feature
-			},
-			ClaimRef:                      nil,
-			PersistentVolumeReclaimPolicy: "",
-			StorageClassName:              volumeStorageClassName,
-			MountOptions:                  nil,
-			VolumeMode:                    nil,
-			NodeAffinity:                  nil,
-		},
-		Status: apiv1.PersistentVolumeStatus{
-			Phase:   "",
-			Message: "",
-			Reason:  "",
-		},
-	}
-
-	volume, err := volumesClient.Create(ctx, &persistentVolumeDefinition, globalCreateOptions)
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "Failed to create volume '%s'", volumeName)
-	}
-	return volume, err
-}
-
 func (manager *KubernetesManager) RemovePersistentVolume(
```

Comment:
shouldn't this one also be removed? if the creation is removed

Reply:
same for `GetPersistentVolume`

Reply:
removed them and a bunch of other code(pre Guillaume!!)